.github/workflows/pipettes.yaml

```diff
@@ -128,10 +128,6 @@ jobs:
         run: cmake --build ./build-host --target pipettes-96-simulator
         working-directory: ot3-firmware
 
-      - name: 'Build 384 simulator'
```

Comment:
A note for both PRs,

I believe some work that @vegano1 was doing included compiling/looking for the 384 firmware. I would just triple check on everything to do w firmware updates and make sure removing the presets etc doesn't break anything.

Reply:
oe-core looks for pioettes-384 hex file to install, but that's not a breaking change. But that's most likely getting changed as part of the oe-core portion of the changes Seth is working on.

Reply:
The nice thing about that is that it's actually all controlled in here by the `subsystem_versions.json` - so it all works still